### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by itanand**
I have added some of the new Resources like Github, and Python . For the DevOps Python and Version Control System is Very Important.

## Changes proposed

In DevOps Roadmap I have make these Changes - I have added some of the new Resources like Github, and Python . For the DevOps Python and Version Control System is Very Important.

## Screenshots

image

## Note to reviewers

In DevOps Roadmap I have make these Changes - > I have added some of the new Resources like Github, and Python . For the DevOps Python and Version Control System is Very **Important.**

### CODE REVIEW

**Review by Sandeep-RN — CHANGES_REQUESTED** [review]
It looks good,
But I would suggest you to move "Version Control System - Git & GitHub" section to after "Operating System Basics" so that it follows an order, new learners will have difficult time understanding Git & GitHub.

and Python can also be moved to after "GoLang" or in between "Operating System Basics" and "Version Control System - Git & Github" this is just optional you can leave it at the top as well I don't think it would make a big difference.

**Comment by itanand** [comment]
Yeah Sure, I can do that

File: DevOps/README.md [hunk 1]
```diff
@@ -1,5 +1,34 @@
 ## DEVOPS
 
+## Version Control System - Git & Github
+
+Automate, customize, and execute your software development workflows right in your repository with GitHub Actions. You can discover, create, and share actions to perform any job you'd like, including CI/CD, and combine actions in a completely customized workflow.
+
+Visit the following resources to learn more:
```
Comment (siddhant-khisty) [inline DevOps/README.md:7]: No need for this intro paragaph

File: DevOps/README.md [hunk 2]
```diff
@@ -1,5 +1,34 @@
...
+
+- [GitHub Awesome Explanation By Kunal Kushwaha ](shorturl.at/bFIT1)
+- [GitHub Actions Documentation](https://docs.github.com/en/actions)
+- [Learn GitHub Actions](https://docs.github.com/en/actions/learn-github-actions)
+- [GitHub Actions - Supercharge your GitHub Flow](https://youtu.be/cP0I9w2coGU)
+- [Automate your Workflow with GitHub Actions](https://www.youtube.com/watch?v=nyKZTKQS_EQ)
```
Comment (siddhant-khisty) [inline DevOps/README.md:13]: Please keep it the exact name of the resource. The resource name is not `GitHub Awesome Explanation` :)

File: DevOps/README.md [hunk 3]
```diff
@@ -1,5 +1,34 @@
...
+
+
+- [Python Website](https://www.python.org/)
+- [Python Getting Started](https://www.python.org/about/gettingstarted/)
+- [Automate the Boring Stuff](https://automatetheboringstuff.com/)
+- [W3Schools - Python Tutorial ](https://www.w3schools.com/python/)
+- [Programmiz - Python ](https://www.programiz.com/python-programming)
+- [Codecademy - Learn Python 3](https://www.codecademy.com/learn/learn-python-3)
+
+
```
Comment (siddhant-khisty) [inline DevOps/README.md:31]: We can exclude python since for cloud-native workloads golang is preferred for it's performance with cloud-based workloads

**Review by siddhant-khisty — CHANGES_REQUESTED** [review]
Suggested a couple of changes. Overall great work.

**Pull request closed without merging**